content/en/hosting/costs/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Hosting Costs"
+linkTitle: "Hosting Costs"
+weight: 2
+aliases:
+
+---
+
+
+## Cost per CHT per Month
+
+In a production environment, the CHT costs $0.10 per user per month to run.  What exactly does this mean? Let's dive into some specifics which importantly cover what is include and excluded and how these costs can vary with each deployment.
+
+{{% pageinfo %}}
+Be sure to read the [Accuracy section](#accuracy) so you understand what the costs on these page mean for your deployment.
+{{% /pageinfo %}}
+
+## How the number was calculated
+
+Medic hosts a number of production CHT instances in Amazon [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS). By using [OpenCost](https://www.opencost.io/), Medic closely monitoring real world costs with actual end users. OpenCost allows us to separate one deployment's CPU, RAM and disk use from another's despite them all running in a multi-tenant EKS cluster.
+
+By looking at dozens of instances and focusing on the highest usage ones, an accurate representation can be derived of a higher cost a self hosting partner.  
+
+Here's some real costs for two week period for two instances that have been renamed to simply **A** and **B**:
+
+| Per Month | A | B | 
+| --- | --- | --- | --- |
+| $/User | $0.05 | $0.10 |  |
+| $ | $94 | $68 |  |
+| Active Users (30 days) | 2000 | 661 |  |
+| DB Size (Millions) | 7.5M | 10.7M |  |
+| DB Growth (Thousands) | 123.2K | 248.0K |  |
+| RAM | $16 | $22 |  |
+| CPU | $6 | $8 |  |
+| Storage | $53 | $24 |  |
+| CPU max used (vCPUs) | 2.9 | 8 |  |
+| RAM max (Gigs) | 20 | 15 |  |
+| Storage Max PVC (TBs) | 0.96 | 0.42 |  |
+
+Deployment **A** has a lot of active users (`2000/mo`), but a slower database growth (`123.2K/mo`) as compared to deployment **B** which has fewer users (`661/mo`) but faster growth (`248K/mo`). These are the types of variations that can cause changes in costs mentioned in the next section.
+
+### Accuracy
+
+As our research was based on live instances, the costs are realistic.  However, since not all users will use the CHT in the exact same way and not all deployments have the same number of users and work flows, these numbers are still subject to some per-deployment fluctuation.
+
+Further, the costs in the table above only cover the variable costs.  That is, the costs incurred of actual usage with in an EKS cluster.  In a multi-tenant deployment where you have maybe three instances (Staging, User Acceptance Testing and Production), this can be critical to know what the three different deployments incurred costs are within your EKS cluster.  
```

Comment:
```suggestion
Further, the costs in the table above only cover the variable costs.  That is, the costs incurred of actual usage with in an EKS cluster.  In a multi-tenant deployment with multiple instances (for example: Staging, User Acceptance Testing and Production), this can be critical to know what the three different deployments incurred costs are within the EKS cluster.  
```